webpack/ForemanTasks/Components/TasksDashboard/Components/TasksCardsGrid/TasksCardsGrid.stories.js

```diff
@@ -37,16 +37,23 @@ storiesOf('TasksDashboard/TasksCardsGrid', module)
     );
 
     return (
-      <TasksCardsGrid
-        time={selectTime}
-        query={{
-          state: selectState,
-          result: selectResult,
-          mode: selectMode,
-          time: selectTime,
-        }}
-        data={object('data', MOCKED_DATA)}
-        updateQuery={action('updateQuery')}
-      />
+      <div>
+        <link
```

Comment:
Do we actually need this `link`?
Can't we add the `patternfly-additions` to https://github.com/theforeman/foreman-tasks/blob/master/webpack/stories/index.scss?

Reply:
added the font awesome to the file instead, thanks!